cmd/integrationArtifactDeploy_test.go

```diff
@@ -39,12 +39,40 @@ func TestRunIntegrationArtifactDeploy(t *testing.T) {
 			Platform:               "cf",
 		}
 
-		httpClient := httpMockCpis{CPIFunction: "IntegrationArtifactDeploy", ResponseBody: ``, TestType: "Positive"}
+		httpClient := httpMockCpis{CPIFunction: "", ResponseBody: ``, TestType: "PositiveAndDeployIntegrationDesigntimeArtifactResBody"}
 
 		err := runIntegrationArtifactDeploy(&config, nil, &httpClient)
 
 		if assert.NoError(t, err) {
 
+			t.Run("check url", func(t *testing.T) {
+				assert.Equal(t, "https://demo/api/v1/IntegrationRuntimeArtifacts('flow1')", httpClient.URL)
+			})
+
+			t.Run("check method", func(t *testing.T) {
+				assert.Equal(t, "GET", httpClient.Method)
+			})
+		}
+	})
+
+	t.Run("Trigger Failure for Integration Flow Deployment", func(t *testing.T) {
```

Comment:
So this test is a negative test but the asserts are all positive, checking that there is no error, checking that the url is correct and checking that a POST request was done. Is there something to check that the trigger actually failed?

Reply:
at deploy level i check **assert.Error(t, err**) which i am injecting via HTTP status code.  this test mainly i use for 3 purpose 
1. injecting deployment error 
2. checking deployment method 
3. checking deployment rest URL